scipy/integrate/quadpack.py

```diff
@@ -386,39 +333,15 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     if not isinstance(args, tuple):
         args = (args,)
 
-    if not vectorized:
-        if norm is not None:
-            raise ValueError("Option 'norm' not available when vectorized=False")
-        if quadrature is not None:
-            raise ValueError("Option 'quadrature' not available when vectorized=False")
-        if workers != 1:
-            raise ValueError("Option 'workers' not available when vectorized=False")
+    # check the limits of integration: \int_a^b, expect a < b
+    flip, a, b = b < a, min(a, b), max(a, b)
 
-        # check the limits of integration: \int_a^b, expect a < b
-        flip, a, b = b < a, min(a, b), max(a, b)
-
-        if weight is None:
-            retval = _quad(func, a, b, args, full_output, epsabs, epsrel, limit,
-                           points)
-        else:
-            retval = _quad_weight(func, a, b, args, full_output, epsabs, epsrel,
-                                  limlst, limit, maxp1, weight, wvar, wopts)
+    if weight is None:
+        retval = _quad(func, a, b, args, full_output, epsabs, epsrel, limit,
+                       points)
     else:
-        if weight is not None:
-            raise ValueError("Option 'weight' not supported when vectorized=True")
-        if args:
```

Comment:
@tupui the args functionality was removed here but not added to the new function `quad_vec`.